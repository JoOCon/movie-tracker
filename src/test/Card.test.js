describe('Card', () => {
  it('should return a type of ADD_MOVIES, with the movies', () => {

  })
})