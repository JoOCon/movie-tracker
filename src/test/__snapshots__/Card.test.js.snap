// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests renders the Card with the correct props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    addFavorite={[Function]}
    favorites={
        Array [
            Object {
              "adult": false,
              "backdrop_path": "/bOGkgRGdhrBYJSLpXaxhXVstddV.jpg",
              "genre_ids": Array [
                12,
                878,
                14,
                28,
              ],
              "id": 299536,
              "original_language": "en",
              "original_title": "The Mockman Prophecy",
              "overview": "This movie is totally fake",
              "popularity": 228.032,
              "poster_path": "/7WsyChQLEftFiDOVTGkv3hFpyyt.jpg",
              "release_date": "2018-04-25",
              "title": "Mock: The Movie",
              "video": false,
              "vote_average": 8.3,
              "vote_count": 7458,
            },
          ]
    }
    movie={
        Object {
            "movie_id": 299536,
            "overview": "As the Avengers and their allies have continued to protect the world from threats too large for any one hero to handle, a new danger has emerged from the cosmic shadows: Thanos. A despot of intergalactic infamy, his goal is to collect all six Infinity Stones, artifacts of unimaginable power, and use them to inflict his twisted will on all of reality. Everything the Avengers have fought for has led up to this moment - the fate of Earth and existence itself has never been more uncertain.",
            "poster_path": "https://image.tmdb.org/t/p/w200/7WsyChQLEftFiDOVTGkv3hFpyyt.jpg",
            "release_date": "2018-04-25",
            "title": "Avengers: Infinity War",
            "vote_average": 8.3,
          }
    }
    removeFavorite={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="favorite-wrapper"
>
          <Route
                    component={[Function]}
                    exact={true}
                    path="/"
          />
          <Route
                    component={[Function]}
                    path="/:user"
          />
          <StarRatings
                    changeRating={null}
                    gradientPathName=""
                    ignoreInlineStyles={false}
                    numberOfStars={5}
                    rating={4.15}
                    starDimension="1em"
                    starEmptyColor="rgb(203, 211, 227)"
                    starHoverColor="rgb(230, 67, 47)"
                    starRatedColor="gold"
                    starSpacing="7px"
                    svgIconPath="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                    svgIconViewBox="0 0 51 48"
                    typeOfWidget="Star"
          />
</div>,
        <img
          alt="movie data"
          className="poster-image"
          src="https://image.tmdb.org/t/p/w200/7WsyChQLEftFiDOVTGkv3hFpyyt.jpg"
/>,
      ],
      "className": "card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Route
              component={[Function]}
              exact={true}
              path="/"
/>,
            <Route
              component={[Function]}
              path="/:user"
/>,
            <StarRatings
              changeRating={null}
              gradientPathName=""
              ignoreInlineStyles={false}
              numberOfStars={5}
              rating={4.15}
              starDimension="1em"
              starEmptyColor="rgb(203, 211, 227)"
              starHoverColor="rgb(230, 67, 47)"
              starRatedColor="gold"
              starSpacing="7px"
              svgIconPath="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
              svgIconViewBox="0 0 51 48"
              typeOfWidget="Star"
/>,
          ],
          "className": "favorite-wrapper",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "exact": true,
              "path": "/",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "path": "/:user",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "changeRating": null,
              "gradientPathName": "",
              "ignoreInlineStyles": false,
              "numberOfStars": 5,
              "rating": 4.15,
              "starDimension": "1em",
              "starEmptyColor": "rgb(203, 211, 227)",
              "starHoverColor": "rgb(230, 67, 47)",
              "starRatedColor": "gold",
              "starSpacing": "7px",
              "svgIconPath": "m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z",
              "svgIconViewBox": "0 0 51 48",
              "typeOfWidget": "Star",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "movie data",
          "className": "poster-image",
          "src": "https://image.tmdb.org/t/p/w200/7WsyChQLEftFiDOVTGkv3hFpyyt.jpg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="favorite-wrapper"
>
            <Route
                        component={[Function]}
                        exact={true}
                        path="/"
            />
            <Route
                        component={[Function]}
                        path="/:user"
            />
            <StarRatings
                        changeRating={null}
                        gradientPathName=""
                        ignoreInlineStyles={false}
                        numberOfStars={5}
                        rating={4.15}
                        starDimension="1em"
                        starEmptyColor="rgb(203, 211, 227)"
                        starHoverColor="rgb(230, 67, 47)"
                        starRatedColor="gold"
                        starSpacing="7px"
                        svgIconPath="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                        svgIconViewBox="0 0 51 48"
                        typeOfWidget="Star"
            />
</div>,
          <img
            alt="movie data"
            className="poster-image"
            src="https://image.tmdb.org/t/p/w200/7WsyChQLEftFiDOVTGkv3hFpyyt.jpg"
/>,
        ],
        "className": "card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <Route
                component={[Function]}
                exact={true}
                path="/"
/>,
              <Route
                component={[Function]}
                path="/:user"
/>,
              <StarRatings
                changeRating={null}
                gradientPathName=""
                ignoreInlineStyles={false}
                numberOfStars={5}
                rating={4.15}
                starDimension="1em"
                starEmptyColor="rgb(203, 211, 227)"
                starHoverColor="rgb(230, 67, 47)"
                starRatedColor="gold"
                starSpacing="7px"
                svgIconPath="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                svgIconViewBox="0 0 51 48"
                typeOfWidget="Star"
/>,
            ],
            "className": "favorite-wrapper",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "exact": true,
                "path": "/",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "path": "/:user",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "changeRating": null,
                "gradientPathName": "",
                "ignoreInlineStyles": false,
                "numberOfStars": 5,
                "rating": 4.15,
                "starDimension": "1em",
                "starEmptyColor": "rgb(203, 211, 227)",
                "starHoverColor": "rgb(230, 67, 47)",
                "starRatedColor": "gold",
                "starSpacing": "7px",
                "svgIconPath": "m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z",
                "svgIconViewBox": "0 0 51 48",
                "typeOfWidget": "Star",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "movie data",
            "className": "poster-image",
            "src": "https://image.tmdb.org/t/p/w200/7WsyChQLEftFiDOVTGkv3hFpyyt.jpg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "disableLifecycleMethods": true,
    "hydrateIn": undefined,
  },
}
`;
